src/js/services/progressbar.js

```diff
@@ -1,4 +1,5 @@
 export const init = () => {
+  document.body.style.backgroundColor = '#fff';
```

Comment:
Why was not a class added with background-color: #fff?

Reply:
styles apply more strongly in css hierarchy.